docs/integrate_gcp_using_google_cloud_shell.md

```diff
@@ -0,0 +1,149 @@
+Integrations. The code leverages our Terraform modules to create resources in you Google Console as well as
+connecting such resources to you Lacework account. To customize these modules look at the [following input
+parameters](https://github.com/lacework/terraform-provisioning/tree/master/gcp#inputs).
+
+**IMPORTANT:** We use the `master` branch in source just as an example. In your code, **do NOT pin to master**
+because there may be breaking changes between releases. Instead we recommend to pin to the release tag (e.g.
+`?ref=tags/v0.1.0`) of one of our [latest releases](https://github.com/lacework/terraform-provisioning/releases).
+
+### For Organization Level Integration
+```hcl
+provider "google" {}
+
+provider "lacework" {}
+
+module "gcp_organization_config" {
+	source          = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/config?ref=master"
+	org_integration = true
+	organization_id = "my-organization-id"
+}
+
+module "gcp_organization_audit_log" {
+	source                       = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/audit_log?ref=master"
+	bucket_force_destroy         = true
+	org_integration              = true
+	use_existing_service_account = true
+	service_account_name         = module.gcp_organization_config.service_account_name
+	service_account_private_key  = module.gcp_organization_config.service_account_private_key
+	organization_id              = "my-organization-id"
+}
+```
+
+__NOTE: Update 'my-organization-id' with your GCP Organization ID. You can use the command `gcloud organizations list` to look up your id.__
+
+### For Project Level Integration
+```hcl
+provider "google" {}
+
+provider "lacework" {}
+
+module "gcp_project_config" {
+	source = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/config?ref=master"
+}
+
+module "gcp_project_audit_log" {
+	source                       = "git::https://github.com/lacework/terraform-provisioning.git//gcp/modules/audit_log?ref=master"
+	bucket_force_destroy         = true
+	use_existing_service_account = true
+	service_account_name         = module.gcp_project_config.service_account_name
+	service_account_private_key  = module.gcp_project_config.service_account_private_key
+}
+```
+
+__NOTE: If you choose the editor, don't forget to save the file.__
+
+### Run the Automation. Run Terraform!
+
+Run the command `terraform init` to download the necessary plugins and modules required to run this automation.
+```
+afiune@cloudshell:~ $ terraform init
+```
+
+Then, run `terraform apply`, this command will create a "plan" of the resources that will be created and stop
+for you to type `yes` to proceed.
+```
+afiune@cloudshell:~ $ terraform apply
+```
+
+![Hit Yes](https://techally-artifacts.s3-us-west-2.amazonaws.com/github-terraform-provisioning-imgs/google-cloud-shell-terraform-apply.png)
+
+**Hit yes!**
```

Comment:
I know I wrote this but I wonder if we should mention that if the automation doesn't work, it is ok to re run terraform apply to continue were it left off, as well as saying that if there are any issues the errors should help troubleshoot the problems. Maaaaybe 🤔

Reply:
I think we should add that as a **NOTE:** @afiune